batchflow/models/metrics/classify.py

```diff
@@ -297,7 +310,11 @@ def specificity(self, *args, **kwargs):
         return self.true_negative_rate(*args, **kwargs)
 
     def prevalence(self, *args, **kwargs):
-        return self._calc_agg(self.condition_positive, self.total_population, *args, **kwargs)
+        """
+        Parameter when_zero doesn't really matter in this case,
+        since total_population is never zero, when targets are not empty.
+        """
```

Comment:
First line of doc should give a quick description of what the method does. Use `Notes` section for implementation/usage details, just like [guide](https://numpydoc.readthedocs.io/en/latest/format.html) prescribes